test/Tabs.js

```diff
@@ -0,0 +1,206 @@
+'use strict';
+
+import dom from 'metal-dom';
+import Tabs from '../src/Tabs';
+
+let tabs;
+
+describe('Tabs', function() {
+	afterEach(function() {
+		if (tabs) {
+			tabs.dispose();
+		}
+	});
+
+	it('should not render anything if there is a tab without label since the validation should fail', () => {
+		tabs = new Tabs({
+			tabs: [
+				{
+					label: 'test_label1'
+				},
+				{
+					label: 'test_label2'
+				},
+				{
+					label: 'undefined'
+				},
+				{
+					label: ''
+				}
+			]
+		});
+
+		let navElement = tabs.element;
+
+		assert.isNull(navElement);
+	});
+
+	it('should render the navigation with the regular style if no type or elementClasses were given', () => {
+		tabs = new Tabs({
+			tabs: [
+				{
+					label: 'test_label1'
+				},
+				{
+					label: 'test_label2'
+				},
+				{
+					label: 'undefined'
+				}
+			]
+		});
+
+		assert.ok(dom.hasClass(tabs.element, 'nav-tabs'));
+	});
+
+	it('should render the navigation with the pills style if pills type was given', () => {
+		tabs = new Tabs({
+			tabs: [
+				{
+					label: 'test_label1'
+				},
+				{
+					label: 'test_label2'
+				},
+				{
+					label: 'undefined'
+				}
+			],
+			type: 'pills'
+		});
+
+		assert.ok(dom.hasClass(tabs.element, 'nav-pills'));
+	});
+
+	it('should fire a "changeRequest" event with correct values if you click on a none active item', () => {
+		tabs = new Tabs({
+			tabs: [
+				{
+					label: 'test_label1'
+				},
+				{
+					label: 'test_label2'
+				},
+				{
+					label: 'undefined'
+				}
+			]
+		});
+
+		let navElement = tabs.element;
+		let pagesNodes = navElement.querySelectorAll('li');
+		let page2 = pagesNodes[1];
+
+		tabs.once('changeRequest', function(event) {
+			assert.ok(!dom.hasClass(page2, 'active'));
+			assert.strictEqual(0, event.lastState.tab);
+			assert.strictEqual(1, event.state.tab);
+			assert.strictEqual(3, event.totalTabs);
+		});
+
+		dom.triggerEvent(page2, 'click');
+	});
+
+	it('should automatically select the next available (not disabled) tab as active if the previous ones are disabled', () => {
+		tabs = new Tabs({
+			tabs: [
+				{
+					label: 'test_label1,',
+					disabled: true
+				},
+				{
+					label: 'test_label2',
+					disabled: true
+				},
+				{
+					label: 'undefined'
+				}
+			]
+		});
+
+		let navElement = tabs.element;
+		let pagesNodes = navElement.querySelectorAll('li');
+		let page = pagesNodes[2];
+
+		assert.ok(dom.hasClass(page, 'active'));
+	});
+
+	it('should set the default value of tab to "4" if there are 3 tabs but the first two is disabled', () => {
```

Comment:
This test description is a bit confusing to me. The test case uses more than 3 tabs, and more than just the first 2 are disabled, so maybe you forgot to update the test's name? It's best to more generic with the test names, you're testing that the next available index will be chosen correctly, not just when there are 3 tabs and 2 disabled.

Also, it seems like this is testing the same case as the previous test, in which case it's best not to have test cases for the same purpose.